src/components/subheader/SubHeader.js

```diff
@@ -0,0 +1,75 @@
+// Libraries
+import React from "react";
+
+// components
+import Description from "./components/description";
+import DiscountSection from './components/discountSection'
+import Loader from "../../commonComponents/loader";
+import ErrorHandler from '../../commonComponents/errorHandler'
+
+// readers
+import subheaderReader from './readers/subheaderReader'
+
+// constants
+import { INITIAL_STATE, subheaderDataUrl } from "./constants/subheader.general";
+
+// helpers
+import fetchData from "../../api/fetch";
```

Comment:
while importing change the name to be more specific to what you're going to fetch